tools/sphinx/source/conf.py

```diff
@@ -66,4 +67,28 @@
     else:
         raise ConfigError(
             f"The environment variable PLANTUML points to a not existing file {plantuml_env}."
-        )
\ No newline at end of file
+        )
+
+def setup(app: any) -> None:
+    """Setup sphinx.
+
+    Args:
+        app (any): The sphinx application.
+    """
+    app.connect('builder-inited', copy_coverage_files)
+
+def copy_coverage_files(app: any) -> None:
```

Comment:
Where is this used ?

Reply:
In line 78. Its required to get all coverage related files copied to the right place, because sphinx can only link to single static html files, but has no inline functionality to grab several files and their assets.